lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -232,6 +232,18 @@ describe('GatherRunner', function() {
       const results = await GatherRunner.run(config.passes, options);
       expect(results.TestedAsMobileDevice).toBe(false);
     });
+
+    it('works when running via DevTools with emulation applied outside of LH', async () => {
+      const driver = fakeDriver;
+      const config = new Config({
+        passes: [],
+        settings: {emulatedFormFactor: 'mobile', deviceScreenEmulationMethod: 'provided'},
+      });
+      const options = {requestedUrl, driver, settings: config.settings};
+
+      const results = await GatherRunner.run(config.passes, options);
+      expect(results.TestedAsMobileDevice).toBe(true);
```

Comment:
this test feels a tad hollow when the `TestedAsMobileDevice` is based on the mocked result of `getBrowserVersion` which is not affected by our `deviceScreenEmulationMethod` here :)

another case for some separate `emulation-test` tests

Reply:
fair. 
new emulation-test added.